nncf/experimental/common/tensor_statistics/collectors.py

```diff
@@ -622,19 +611,17 @@ def _register_reduced_input_impl(self, x: NNCFTensor) -> None:
         else:
             reduced = x
         if 0 in self._aggregation_axes:
-            if self._container:
-                reduced = self._aggregation_fn(
-                    self._tensor_processor.stack([reduced, self._container]), axis=0, keepdims=False
-                )
-            self._container = reduced
+            stacked_tensors = self._tensor_processor.stack([reduced, *self._container], axis=0)
+            aggregated = self._aggregation_fn(stacked_tensors, axis=0, keepdims=self._keepdims)
+            aggregated = self._tensor_processor.squeeze(aggregated, 0)
+            self._container = [aggregated]
```

Comment:
Could you please provide the example for the container size/shape transformation?